docs/src/pages/versions/StableVersions.js

```diff
@@ -43,42 +45,51 @@ async function getBranches() {
 
 class StableVersions extends React.Component {
   state = {
-    versions: [],
+    docs: [],
   };
 
   componentDidMount = async () => {
     const branches = await getBranches();
-    const versions = branches.map(n => n.name);
-    versions.reverse().pop(); // most recent first & remove 'latest'
-    versions.push('v0.20.1');
-
-    this.setState({ versions });
+    let docs = branches.map(n => n.name);
+    docs.sort().reverse();
+    docs.pop(); // most recent first & remove 'latest'
+    docs = docs.map(version => {
+      return {
+        version,
+        // Replace dot with dashes for Netlify branch subdomains
+        url: `https://${version.replace(/\./g, '-')}.material-ui.com`,
+      };
+    });
+    docs.push({
+      version: `v${process.env.LIB_VERSION}`,
+      url: document.location.origin,
+    });
+    docs.push({
+      version: 'v0.20.1',
+      url: 'https://v0.material-ui.com',
+    });
+    docs = uniqBy(docs, 'version');
+    docs[0].url = 'https://material-ui.com';
+    this.setState({ docs });
   };
```

Comment:
Why not just add the current version to the versions array if it isn't already present (i.e if there isn't a version specific branch on GitHub)? Job done.

Reply:
It's what the `uniq` is doing without the need for a condition.

Reply:
I meant instead of the entire  block of code, as well as not needing the dependancy.  The rest is (was) already taken care of, although I appreciate the desire to move the logic out of the render code.

Reply:
```js
let versions = [ 'v1.2.0', 'v1.1.0', 'v1.0.0' ]
> [ 'v1.2.0', 'v1.1.0', 'v1.0.0' ]

versions.unshift('v1.2.0')
> [ 'v1.2.0', 'v1.2.0', 'v1.1.0', 'v1.0.0' ]

versions =  [...new Set(versions)]
> [ 'v1.2.0', 'v1.1.0', 'v1.0.0' ]